Blog/view/logInConfirmation.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+session_start();
```

Comment:
This should go in your `include_all` file so the session is always started by default.